src/lib/utils/Mark.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import Dispatch from "./Dispatch";
+import uuid from "uuid";
+
+const Mark = (mark, inner, props) => {
+  const Dispatcher = Dispatch(mark.type);
+  return (
+    <Dispatcher
+      key={uuid()}
+      nodeData={mark}
```

Comment:
Would markData be clearer? Or, alternatively I kind of liked how you passed in the attrs themselves as a prop before. I don't think the mark handlers need any other mark info beside attrs.

Reply:
Addressed in e68239b62b83bd87

Reply:
Ooh, the only caution I have there is what happens if a mark gets an attr of key or inner?

Reply:
key is kind of informally namespace for React   should I throw an Error "Illegal attribute" 

don't really have another idea ...

Reply:
Addressed in ae6bf92